src/form-data-wrapper.js

```diff
@@ -76,7 +77,7 @@ export class FormDataWrapper {
    * @param {string} value The field's value.
    */
   append(name, value) {
-    if (!this.formData_.entries) {
+    if (!this.formData_['entries']) {
```

Comment:
Why is this one bracketed, but the others aren't?

Reply:
Made all others bracketed for consistency. Not sure why the type checker doesn't complain about the other ones...